source/App.jsx

```diff
@@ -12,9 +13,10 @@ const App = () => (
     <Switch>
       <Route exact path="/" component={Home} />
       <Route path="/sponsor" component={Sponsor} />
+      <Route path="/404" component={Internal_Brian} />
       <Route component={() => <Error message="404 Not Found" />} />
     </Switch>
   </BrowserRouter>
 );
 
-export default App;
+export default App;
```

Comment:
Looks like you removed the newline at the end of the file. It's generally good to leave the newline there, feel free to Google why this is done.